src/main/java/uk/gov/hmcts/reform/ccd/document/am/model/StoredDocumentHalResource.java

```diff
@@ -6,21 +6,29 @@
 import java.util.List;
```

Comment:
Can we replace all getters, setters and other constructor methods with Lombok annotation?

Reply:
I have done.